multiqc/modules/rseqc/read_distribution.py

```diff
@@ -50,6 +50,9 @@ def parse_reports(self):
                 d['{}_tag_count'.format(k)] = int(r_search.group(2))
                 d['{}_tags_kb'.format(k)] = float(r_search.group(2))
 
+        d['other_intergenic_tag_count'] = d['total_tags']-d['total_assigned_tags']
+        d['other_intergenic_total_bases'] = 0
```

Comment:
What's this line used for?

Reply:
Sorry for the late reply. Do you mean the row
``d['other_intergenic_total_bases'] = 0``
?
Probably I just inserted it trying to be consistent with other record that have *_tag_count, *_total_bases and *_tags_kb.

Reply:
Yup - ok I see. The variable wasn't been accessed or written anywhere that I could see (from memory I think the others are set and then incremented at a later stage?). Removing it didn't seem to have any effect, and I figure it's better to have cleaner and clearer code 👍